iOS/rainbow/Controller/Booklet/BookletViewController.swift

```diff
@@ -138,25 +173,44 @@ class BookletViewController: UIViewController, UIPageViewControllerDataSource {
                 pageItemController.titleString = pages[itemIndex].title
                 pageItemController.statementString = pages[itemIndex].description
                 pageItemController.linkString = pages[itemIndex].link
-                guard let image = UIImage(named: pages[itemIndex].imagePath) else {
-                    return pageItemController
-                }
-                pageItemController.image = image
+                pageItemController.image = getBookletImage(page:pages[itemIndex])
+                
                 return pageItemController
+                
             default:
                 guard let pageItemController = self.storyboard?.instantiateViewController(withIdentifier: "CoverController") as? BookletCoverController else {
                     return nil
                 }
                 pageItemController.itemIndex = itemIndex
                 pageItemController.titleString = pages[itemIndex].title
                 pageItemController.subTitleString = pages[itemIndex].subtitle
+                pageItemController.image = getBookletImage(page:pages[itemIndex])
+                
                 return pageItemController
             }
         }
         
         return nil
     }
     
+    func getBookletImage(page:Page) -> UIImage{
+        
+        if( page.imagePath == "" ){
+            
+            let url = URL(string: page.imageURL )
+            
+            guard let data = try? Data(contentsOf: url!) else {
+                print("There was an error!")
+                return UIImage()
+            }
+            return UIImage(data: data)!
+        }else{ guard UIImage(named: page.imagePath) != nil else {
+            return UIImage()
+            }
+            return UIImage(named: page.imagePath)!
```

Comment:
Just a friendly reminder for everyone that using exclamation marks to unwrap an object that could potentially be nil can easily result in a fatal exception. I'll swoop back around and update this, as well as Swiftlint, but please try to be mindful of using the `!` in swift when you're not doing equality checks.